### PR TITLE
Edited n00b-GettingStarted.md and smaller edits to README.md and n00b-overview.md

### PR DESCRIPTION
**Pull request by NeilCuzon**
Hi, I love your program and how it creates diagrams and graphs using text. but I had some small trouble in initially understanding the product's use.. 

I updated and expanded documentation for noob-gettingStarted.md.
-made some alterations with the instructions in effort to make it more n00b friendly.
-added a new screenshot and edited directions for copying markdown code from live editor
-edited instructions for deploying mermaid with html.
-added link to Atom-mermaid plugin

I also made a few changes in README.md, for the same objective of making it more noob-friendly. 

More Power. 
-Neil

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,12 +6,14 @@
...
 <!-- <Main description> -->
-__Generate diagrams, charts, graphs or flows from markdown-like text via javascript.__
-See our [documentation](http://mermaid-js.github.io/mermaid/) and start simplifying yours. Play in our [live editor](https://mermaidjs.github.io/mermaid-live-editor/) or jump straight to the [installation and usage](http://mermaid-js.github.io/mermaid/#/usage).
+__mermaid is a Javascript based diagramming and charting tool. It generates diagrams flowcharts and more, using markdown-inspired text for ease and speed.__
+
+For more information and help in gettin started, please view our [documentation](http://mermaid-js.github.io/mermaid/) and start simplifying yours. Play with our [live editor](https://mermaidjs.github.io/mermaid-live-editor/) or jump straight to the [installation and usage](http://mermaid-js.github.io/mermaid/#/usage).
```
Comment (jgreywolf) [inline README.md]: typo in __gettin started__

Comment (NeilCuzon) [reply README.md]: noted, will correct it immediately.

File: docs/n00b-overview.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Overview for n00bs
 
-As a sysadmin I frequently have to document things, including drawing stuff.
+mermaid is a tool that aims to make diagrams and flowcharts for documentation, easier. 
 
-Using mermaid, I can type this as a comment in a script:
+with mermaid, diagramscan be created through comments like this in a script:
```
Comment (mvandermade) [inline docs/n00b-overview.md]: diagrams can (missing space)

File: docs/n00b-overview.md [hunk 3]
```diff
@@ -11,17 +11,19 @@ B --> C[Server01]
...
+Since the diagram source is text based, it can be part of production scripts (and other pieces of code). So less time needs be spent on documenting as a separate task.
 
-Comparing with Visio and similar applications, mermaid is a really fast way to create good visualizations. This is especially apparent when editing a complex visualisation, this could take me hours in a desktop application but takes minutes (or even less if generation has been scripted) with mermaid.
-
-With mermaid I can spend a fraction of the time I normally would spend, and instead automate the diagram generation and end up saving even more time. I love it!
+Comparing with Visio and similar applications, mermaid is a really fast way to create good visualizations. This is especially apparent when editing a complex visualisations, a process that usually takes hours in a desktop application, but only takes minutes (or even less if generation has been scripted) with mermaid.
 
+mermaid can potentially cut down the amount of time and effort spent on the process of creating diagrams, to a fraction of what you sually put in.
```
Comment (mvandermade) [inline docs/n00b-overview.md]: sually -> usually

File: docs/n00b-overview.md [hunk 4]
```diff
@@ -11,17 +11,19 @@ B --> C[Server01]
...
-
-With mermaid I can spend a fraction of the time I normally would spend, and instead automate the diagram generation and end up saving even more time. I love it!
+Comparing with Visio and similar applications, mermaid is a really fast way to create good visualizations. This is especially apparent when editing a complex visualisations, a process that usually takes hours in a desktop application, but only takes minutes (or even less if generation has been scripted) with mermaid.
 
+mermaid can potentially cut down the amount of time and effort spent on the process of creating diagrams, to a fraction of what you sually put in.  
 
 However, a lot of the mermaid documentation is geared to professional frontend developers, presuming a skill set which I simply do not have.
 
-I needed a really basic instruction. And here it is.
+If you need some basic instructions and introductions. And here are a few good places to start:
```
Comment (mvandermade) [inline docs/n00b-overview.md]: I would personally remove . And and place a comma.
If you need some basic instructions and introductions, here are a few good places to start:

File: docs/n00b-gettingStarted.md [hunk 5]
```diff
@@ -1,42 +1,47 @@
...
+
+It is also an easier way to develop diagrams. You can also click "Copy Markdown" to copy the markdown code for the diagram, that can then be pasted directly into your documentation. 
+
+![Flowchart](./img/liveEditorOptions.png)
 
 The `Mermaid configuration` is for controlling mermaid behaviour. An easy introduction to mermaid configuration is found in the [Advanced usage](n00b-advanced.md) section. A complete configuration reference cataloguing default values is found on the [mermaidAPI](mermaidAPI.md) page.
 
 
-## mermaid using plugins
+## 2. Using mermaid plugins:
```
Comment (mvandermade) [inline docs/n00b-gettingStarted.md]: Maybe refer for the full list to :  docs/integrations.md
https://github.com/mermaid-js/mermaid/pull/1278/files (for a bigger list)

Comment (erelling) [reply docs/n00b-gettingStarted.md]: Excellent, just learning how to incorporate your suggestion...

File: docs/n00b-overview.md (hunk 3 above)
Comment (mvandermade) [inline docs/n00b-overview.md]:
```suggestion
mermaid can potentially cut down the amount of time and effort spent on the process of creating diagrams, to a fraction of what you usually put in.  
```

Comment (mvandermade) [reply docs/n00b-overview.md]: @erelling this is also a nice feature of github :) I can propose a suggestion using the 
image button

**Comment by NeilCuzon** [comment]
thank you for the reviews, I'll correct them immediately.

File: docs/n00b-gettingStarted.md (hunk 5 above)
Comment (NeilCuzon) [reply docs/n00b-gettingStarted.md]: added, the link to integrations. thanks for the feedback.

Comment (NeilCuzon) [reply docs/n00b-gettingStarted.md]: thank you for the feedback!

**Comment by NeilCuzon** [comment]
thanks for the reviews and the edits! this is my first time contributing to Github and I'm thankful for the guidance.

**Comment by erelling** [comment]
😃First timers you and me both!

**Comment by mvandermade** [comment]
Good to hear! Have fun :)

**Pull request merged**